app/models/course/assessment/question/programming.rb

```diff
@@ -110,22 +110,40 @@ def non_autograded_template_files=(template_files)
 
   private
 
+  # Create new package or re-evaluate the old package.
+  def process_package
+    return if @duplicating
```

Comment:
I have defined a new `skip_process_package?` method instead, cause I need to override it in factories.